### PR TITLE
Broken links

### PR DESCRIPTION
**Pull request by LornaSIL**
Trying to fix broken links.

### CODE REVIEW

File: legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm [hunk 1]
```diff
@@ -2139,7 +2139,7 @@ <h3>Punctuation</h3>
 <div id="Troubleshooting">
 <h2>Troubleshooting</h2>
 
-<p>For any other questions, <a target="_blank" href="http://www.tavultesoft.com/contact.php">contact us</a>.</p>
+<p>For any other questions, <a target="_blank" href="https://help.keyman.com">contact us</a>.</p>
```
Comment (darcywong00) [inline legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm]: Maybe we should use the community site [link](https://community.software.sil.org/c/keyman) for all these contact links.

@mcdurdin recently removed the email links in sillsdev/keyman-help.keyman.com#186

File: legacy/k/korean_rr/source/help/korean_rr.php [hunk 2]
```diff
@@ -58,11 +58,11 @@
...
-combined to form <a target="_blank" href="http://en.wikipedia.org/wiki/Hangul">hangeul</a> characters by pressing the space bar, or <a target="_blank" href="http://en.wikipedia.org/wiki/Hanja">hanja</a> ideographs by selecting from an IME (Input Method Editor) list.   
+This keyboard is designed for entering the <a target="_blank" href="https://en.wikipedia.org/wiki/Korean">Korean</a> language, supporting both the 
+<a target="_blank" href="https://en.wikipedia.org/wiki/Korean_language#Writing_system">Hangeul</a> alphabet and 
+<a target="_blank" href="https://en.wikipedia.org/wiki/Hanja">Hanja</a> ideographic characters, as used in Republic of Korea.  
+Korean text is entered using Latin alphabet characters, closely following the Korean standard Revised Romanization conventions and displayed as <a target="_blank" href="https://en.wikipedia.org/wiki/Hangul_consonant_and_vowel_tables#jamo">jamo</a> character components, which are
```
Comment (darcywong00) [inline legacy/k/korean_rr/source/help/korean_rr.php]: I think you can drop the `#jamo` portion of the link. I didn't see that anchor on the page

File: legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm [hunk 3]
```diff
@@ -2148,22 +2148,22 @@ <h2>Further Resources</h2>
 <div id="Related">
 <h3>Related Keyboard Layouts</h3>
 <p>Other Galaxie BibleScript Greek keyboards:</p>
-<a href='http://keymankeyboards.com/?id=413'>Galaxie BibleScript Greek and Hebrew</a>
+<a href='https://keyman.com/keyboards/galaxiegreekandhebrew'>Galaxie BibleScript Greek and Hebrew</a>
 
 <p>Other Classical and Polytonic Greek keyboard layouts:</p>
 <p><a href="http://www.tavultesoft.com/greek/" target="_blank">Greek Keyboards Home</a></p>
```
Comment (darcywong00) [inline legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm]: Do the tavultesoft links change? Something like
https://keyman.com/keyboards?q=greek

Comment (mcdurdin) [reply legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm]: Use https://keyman.com/greek. It forwards to http://tavultesoft.com/greek for now but won't in the future.

File: legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm (hunk 1 above)
Comment (mcdurdin) [reply legacy/g/galaxiebiblescriptmnemonic/source/help/welcome-greek.htm]: For contact links, you can use https://keyman.com/contact for general links or https://community.software.sil.org/c/keyman for support forum ("Keyman Community")

**Pull request merged**